### ISSUE TITLE
Thick supply lines should be optional

### ISSUE THREAD

**Issue by MXB2001** [freeorion/freeorion#4574]
They're horrible for new players since they scream out at them saying look at me but are almost irrevelant. Either an option to toggle them on when needed or an automatic routine that only turns them on when they actually mean something.

From what I've gathered now they indicate unused PP for empires with isolated (from supply) systems. Only useful in that (apparently) rare case. The program should be able to tell if there are isolated systems and then turn on the display to remind the player of the fact.

**Comment by Oberlus** [comment]
I agree. Simple solution could be making transparent all the thick lines unless there is PP surplus in that supply group.

But I still would like the drawing of the thick lines more consistent than currently: now they show no fathomable criteria to whether plotting thick or thin lines. That's annoying to some players.
Regarding this, and complementary to this issue, plotting algorithm could just plot all supply lines thick (but the outer part transparent unless there is PP surplus).

**Comment by MXB2001** [comment]
Just a note that I discovered a way to remove the thick lines (which look bad due to a bug and don't meet up) in the config. Set core starlane multiplier to 1 and set the starlane thickness to 0.2 (alternately one can set it to 10 and all starlanes will be wide and fat) but any settings in the middle around 5 will (in spite of the multi being 1) cause the broken looking core lanes to appear.
